.gitignore

```diff
@@ -49,6 +49,8 @@ projects/*
 !projects/*.*
 !projects/Makefile
 .venv
+deps-install
+deps-download
```

Comment:
I hope this path will work work Prestissimo later and DuckDB where there is some issue if DuckDB is in the presto/presto-native-execution path. Maybe we have to adjust more for that once this gets added to prestissimo.

Reply:
I tested this with Prestissimo and it works. Let's add to the README after we advance Velox with this change.

Reply:
I clarified in the README that a common directory is preferred if we want to reuse for Prestissimo.

Reply:
For Prestissimo only users, it would default to `presto/presto-native-execution/deps-download` and `presto/presto-native-execution/deps-install`

Reply:
^^ so something we need to add to the presto .gitignore when velox is updated there.